src/portfft/dispatcher/subgroup_dispatcher.hpp

```diff
@@ -143,6 +142,11 @@ PORTFFT_INLINE void subgroup_impl(const T* input, T* output, const T* input_imag
   IdxGlobal rounded_up_n_ffts = round_up_to_multiple(n_transforms, static_cast<IdxGlobal>(n_ffts_per_wg)) +
                                 (subgroup_local_id >= max_wis_working);
 
+  const bool input_batch_interleaved = input_stride == n_transforms && input_distance == 1;
+  const bool output_batch_interleaved = output_stride == n_transforms && output_distance == 1;
+  const bool is_packed_input = input_stride == 1 && input_distance == fft_size;
+  const bool is_packed_output = output_stride == 1 && output_distance == fft_size;
```

Comment:
Could you make the variable names consistent? I like "is input/output packed/batch interleaved".

Reply:
Good idea! Done in https://github.com/codeplaysoftware/portFFT/pull/142/commits/ca5742962ed755442d1e2af7b39c0f7876386ab5